recipe/meta.yaml

```
@@ -11,30 +11,47 @@ source:
   sha256: "{{ sha256 }}"
 
 build:
+  # No mongodb on s390x
+  skip: True  # [py<34 or s390x]
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-build-isolation -vv
 
 requirements:
   host:
     - pip
     - python
+    - setuptools
+    - wheel
   run:
     - python
```

```suggestion
    - python
    - pymongo >=3
```
https://github.com/mongodb-labs/mongo-mockup-db/blob/1.8.1/setup.py#L31C5-L31C21
Or am I missing something?

---

That is... an _excellent_ question.  I wonder if I put it in `test.requires` because I could only see it being used in the tests... 🤔